xdsl/irdl/operations.py

```diff
@@ -870,7 +872,7 @@ def lazy_traits_def(future_traits: Callable[[], tuple[OpTrait, ...]]):
 _OPERATION_DICT_KEYS = {key for cls in Operation.mro()[:-1] for key in cls.__dict__}
 
 
-def _is_const_classvar(field_name: str, annotation: Any) -> bool:
+def is_const_classvar(field_name: str, annotation: Any) -> bool:
```

Comment:
utils? also we should probably reuse for attributes if we're making this one public

Reply:
Sure, should I make a new PR?